conans/test/command/download_test.py

```diff
@@ -0,0 +1,38 @@
+import unittest
+import os
+from conans.test.utils.tools import TestClient, TestServer
+from conans.test.utils.cpp_test_files import cpp_hello_conan_files
+from conans.model.ref import ConanFileReference
+
+
+class DownloadTest(unittest.TestCase):
+
+    def download_reference_without_packages_test(self):
+        client = TestClient()
+        conanfile = """from conans import ConanFile
+class Pkg(ConanFile):
+    settings = "os"
+"""
+        client.save({"conanfile.py": conanfile})
+
+        server = TestServer()
+        servers = {"default": server}
+
+        client = TestClient(servers=servers, users={"default": [("lasote", "mypass")]})
+
+        files = cpp_hello_conan_files()
```

Comment:
I think you don't need real cpp packages, with the conanfile you are good.

Reply:
Yes, exactly, this test can be simplified.